tests/test_import_export.py

```diff
@@ -17,6 +17,20 @@
 from tests.data import expected as expected_data, input as input_data
 
 
+def _normalize_xml(dirname):
+    """Removes whitespace from xml files in the given directory."""
+    for dname, _, files in os.walk(dirname):
+        for fname in files:
+            fpath = os.path.join(dname, fname)
+            if not fpath.endswith('.xml'):
+                continue
+            with open(fpath) as f:
+                s = f.read()
+                s = clean_xml(s)
+            with open(fpath, 'w') as f:
+                f.write(str(s))
```

Comment:
Instead of this, can you add `encoding="UTF-8"` inside `etree.tostring` in `clean_xml`?